mdesnoyer/wonderland
# Keypresses (left arrow, right arrow and Escape) got messed up (Image Zoom AND Paging results) #1324

# Changes
- added `handleKeyEvent` function to `Thumbnails.js` so we can deal with any key presses that are initiated in `ZoomThumbnail`
- make sure (in `ZoomThumbnail) that we store any existing
  `document.body.onkeydown` event(s) before we usurp them for our own evil
- use `self` rather than `this`
- added `ThumbnailOverlay` component to handle the keyup stuff (called in `Thumbnails`)
- removed keydown stuff from `ZoomThumbnail` as we were getting event confusion with multiple components
- updated some translations
# Test Plan
- Results page
  - ensure left and right arrows work as expected, previous and next.
- Image Zoom
  - ensure left and right arrows work as expect
  - ensure escape kills the modal (Image Zoom)
- Bonus Points
  - make sure that the Results still work after being in Image Zoom (since they are hooked to the same handler)
